includes/learndash/class-accredible-learndash-learndash-functions.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Accredible LearnDash Add-on LearnDash functions class
+ *
+ * @package Accredible_Learndash_Add_On
+ */
+
+defined( 'ABSPATH' ) || die;
+
+if ( ! class_exists( 'Accredible_Learndash_Learndash_Functions' ) ) :
+	/**
+	 * Accredible LearnDash Add-on LearnDash functions class.
+	 */
+	class Accredible_Learndash_Learndash_Functions {
```

Comment:
This class is just required to stub LearnDash built-in functions in unit testing.